### ISSUE TITLE
API Catalogue

### ISSUE THREAD

**Issue by gmuratk** [camaraproject/WorkingGroups#258]
Inspired by the API Backlog Table, it may be good to have a table format that serves as the catalogue of CAMARA APIs Operations. 

As an example format:

<html>
<body>

API FAMILY | API | version | Operation | Path | Description
-- | -- | -- | -- | -- | --
[QualityOnDemand](https://github.com/camaraproject/QualityOnDemand) | [qod-api.yaml](https://github.com/camaraproject/QualityOnDemand/blob/release-0.8.1/code/API_definitions/qod-api.yaml) | 0.8.1 | POST | /sessions | Create QoS Session to manage latency/throughput priorities
QualityOnDemand | qod-api.yaml | 0.8.1 | GET | /sessions/{sessionId} | Querying for QoS session resource information details
QualityOnDemand | qod-api.yaml | 0.8.1 | DELETE | /sessions/{sessionId} | Deleting a QoS session
QualityOnDemand | qod-api.yaml | 0.8.1 | POST | /notifications |  
DeviceStatus | device-status.yaml | 0.5.0-wip | POST | /roaming | Get the current roaming status and the country information
DeviceLocation | location.yaml | 0.2.0-wip | POST | /verifications | Execute location verification for a user equipment

</body>
</html>

**Comment by hdamker** [comment]
I'm not sure about this idea (where and how to maintain and keep current), but just a remark regarding the example:


QualityOnDemand | qod-api.yaml | 0.8.1 | POST | /notifications
-- | -- | -- | -- | --

This entry shouldn't be in the list as it is only the callback from server to client in case of a notification. Within 0.9.0 version of the API this be clearer, as it won't be on the top-level any more.

**Comment by gmuratk** [comment]
@hdamker I agree with your concern around maintenance.  If there is agreement for the API Backlog table to be maintained with all the current columns, then maybe we can just add the list/description of 'Operations' as a new column to the API Backlog table.  That will give the same result from my perspective.

**Comment by shilpa-padgaonkar** [comment]
https://github.com/camaraproject/WorkingGroups/discussions/260
